learn/wavemaker-release-notes/v10-5-6.md

```diff
@@ -18,19 +18,14 @@ sidebar_label: "Release v10.5.6"
 
 - Fixed issue related to exporting project shell to EDN.
 - Fixed issues related to editing and displaying the user profile name.
-- Fixed issue related to default messages which are not localized.
-- Fixed issue related to the **onSuccess** handler for file upload variable to customize the upload success notification.
-- Fixed issue related to the `reset()` method that was not working for an HTML upload widget.
+- Few messages which were not localized are now localized.
+- When the **onSuccess** handler for a service variable used to upload file is overridden, the default success toaster is now suppressed. This allows users to customize the behavior as per the need.
+- On clicking reset on a Form, the File widget inside it would not reset. This issue is now fixed
 - Fixed issue related to accessing the second level Partial page widgets from `App.activePage`.
 - Fixed issue related to the **Animation Interval** property for the **Carousel** widget to set it through the script.
 - Fixed issue related to the **Chips** widget dataset that is not updating the UI when the variable is updated.
-
-### Data Table
-
-- Fixed issue related to DataTable column styles that are given in %.
-- Fixed issue related to the **INFO** toaster for Inline Edit in the DataTable row.
-- Fixed issues related to applying stypes for Data Table, including "table-bordered" and "table-striped" not migrating from 9.x to 10.x version.
-- Fixed issue related to the pagination based on the **Records per Request** property in the Data Table.
+- Fixed scrolling issues with content inside a design dialog.
```

Comment:
These two should go under Bug Fixes. They are not related to Table widget

Reply:
@vibhus which two?

Reply:
OK, I misunderstood the change. Looks fine.